transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/JSONPage.java

```diff
@@ -127,7 +127,7 @@ public void handleValueChange(ValueChangeEvent event) {
                             return;
                         }
                         String jsonText = getJsonText((IFile) resource);
-                        if (!Util.isJSONValid(jsonText)) {
+                        if (!Util.jsonValid(jsonText)) {
```

Comment:
why do u remove the "is" in getter method names? imho using "is" as prefix is the standard way for booleans.